### PR TITLE
@eessex => Social embed

### PR DESCRIPTION
**Pull request by kanaabe**
This adds `jsonp` and `react-oembed-container` packages. In an earlier implementation I started writing the jsonp fetcher and script injection from scratch but in the end these libs were way less flaky and covered more edge cases. If anyone is weary about adding more dependency load let's talk --
 these are both pretty small libs tho. 

- [x] Tests
- [x] NewsArticle integration

See notes here: https://artsyproduct.atlassian.net/browse/GROWTH-268

social-embeds

### CODE REVIEW

File: src/Components/Publishing/Sections/SocialEmbed.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  }
+
+  getEmbedUrl = () => {
+    const { url } = this.props.section
+
+    if (url.match("twitter")) {
+      return TWITTER_EMBED_URL + `?url=${url}`
+    } else if (url.match("insta")) {
+      return INSTAGRAM_EMBED_URL + `?url=${url}`
+    }
```
Comment (eessex) [inline src/Components/Publishing/Sections/SocialEmbed.tsx]: This react-oembed-container library is awesome, can't believe how compact this is!
